content/blog/gdpr-cheat-sheet.en.md

```diff
@@ -0,0 +1,139 @@
+
+For you as a consumer, they are here to help. If you think that a controller is not processing your data correctly, you can contact the data protection supervisory authorities free of charge at any time.
+
+If you are not sure which authority you should contact, take a look at our [data protection supervisory authorities finder]({{< ref "supervisory-authorities" >}}#finder).
+
+### Data Processor
+
+A data processor is - as the name implies - a company that processes personal data on behalf of a [data controller](#data-controller). For example, if a company's email server hosted externally, the host acts as the data processor.
+
+The data controller and the data processor enter into a so-called "commissioned data processing agreement" (in German, this is _Auftragsdatenverarbeitungs-Vertrag_, or _ADV-Vertrag_ for short), which regulates exactly how the data is processed.
+
+What is important for you as a consumer: As a rule, the data processor is not the person to look for if you want to exercise your [rights](#rights). Instead, contact the data controller directly.
+
+### Special Categories of Personal Data
+
+Some data is more sensitive than others. While you may have no qualms about giving your name to a new app, it doesn't have to know your religious beliefs. The GDPR recognizes that these so-called "special categories of personal data" deserve and need special protection. Therefore, the [processing](#processing) of such data is only allowed under certain circumstances.
+
+Concretely, this concerns the following data:
+
+* Racial or ethnic origin
+* Political opinions
+* Religious or philosophical beliefs
+* Trade union membership
+* Genetic data
+* Biometric data for the purpose of uniquely identifying a natural person
+* Data concerning health
+* Data concerning a natural person’s sex life or sexual orientation
+
+### Data subject
+
+The data subject is the person whose [personal data](#personal-data) are [processed](#processing). As a consumer, this usually refers to you.
+
+### Data Minimisation
+
+Data minimisation is an important principle for data protection that the GDPR prescribes. The idea behind it is simple: Only the data necessary for carrying out the stated purpose should be collected and [processed](#processing). If a company wants to send you a newsletter, all it needs is your email address. Following the data minimisation principle, it should not collect any other data.
+
+This concept is important not least because of security reasons. We live in a time where users' private data are regularly leaked to the public because companies have not sufficiently secured them. Data that a company does not have cannot be lost.
+
+### Data Protection Officer
+
+A data protection officer's task is to ensure that a company (or another organisation, such as an authority) complies with data protection law. He is appointed by the company itself and must have a certain degree of independence in order to perform his duties.
```

Comment:
The term `authority` may be a little misleading here. We usually use `public body` (although technically, that has a bit of a different meaning). `Government agency` also works fine.

Reply:
I went to look at how it was phrased and [here](https://gdpr-info.eu/art-37-gdpr/), it uses `public authority`. I did more research and it seems that the GDPR itself does not specify what this entails, and it is up to member states to define it exactly.

For the UK, it's broader than government agencies only, and using that might narrow the definition too much. Would it be too technical to leave it as `public authority`?

Reply:
`public authority` is definitely fine, too. I just want to avoid confusion with the supervisory authorities (although, to be fair, they will have their own DPOs, too).

Reply:
I decided to use `government agency` in the end, since, after all, it's meant to be one example. This would definitely make a clearer distinction from supervisory authorities.